tests/test_observables.py

```diff
@@ -461,6 +473,17 @@ def test_robtex_fdns(self, mock_get=None, mock_post=None):
         ).start()
         self.assertEqual(report.get("success", False), True)
 
+    def test_dnstwist(self, mock_get=None, mock_post=None):
```

Comment:
to avoid duplication of code could you please move this code only on `utils.py` under the `CommonTestCases_url_domain` class? In this way you just need to write the test once